packages/cli/src/ActiveWorkflowRunner.ts

```diff
@@ -350,6 +350,10 @@ export class ActiveWorkflowRunner implements IWebhookManager {
 		const webhooks = WebhookHelpers.getWorkflowWebhooks(workflow, additionalData, undefined, true);
 		let path = '';
 
+		if (webhooks.length === 0) return;
```

Comment:
This return statement would cause the cache to not be populated. Don't we want this to happen?

Reply:
If a workflow has no webhooks to add, no new webhooks will be added to the `webhook_entity` table, so `populateCache` will not find anything new to add to the cache. Please let me know if I'm misunderstanding this.